app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -2389,6 +2394,35 @@ private void setShuffleButton(@NonNull final ImageButton button, final boolean s
 
 
 
+    /*//////////////////////////////////////////////////////////////////////////
+    // Playlist append
+    //////////////////////////////////////////////////////////////////////////*/
```

Comment:
Does this have to be moved here into the player?
The player is already bloated as is and does not need more functionality, that it is not supposed to have.

Reply:
It is needed for appending from the Main player, and I just removed it from PlayQueueActivity to remove duplicated code.

Reply:
Yeah, I don't like it but nothing that can be done about it here.
I have #8011 in which I will hopefully be able to address it